hackathon/templates/hackathon/hackathon-view.html

```diff
@@ -0,0 +1,157 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+    <!-- Render Hackathon details & Registration button -->
+    <div class="row">
+        <div class="col-md-12 col-lg-4">
+
+            <div class="container">
+                <div class="row">
+                    <div class="col p-0">
+                        <img width="100%" 
```

Comment:
Yep, moved the style to `styles.css`.

Reply:
Regarding the self-closing tag, as far as I remember the slash at the end should be set. Do you have some documentation that this is no longer good practice? Just curious to see if there is any mention of why.

Reply:
Changed the image as well.

Reply:
@stefdworschak 
Best answer to this would be from here:
https://stackoverflow.com/a/3558200
It's XHTML, but normal HTML5 elements shouldn't require them. It's syntactic sugar ([in case others are reading this](https://en.wikipedia.org/wiki/Syntactic_sugar)), essentially, for people coming from XHTML backgrounds.
Can simply be ignored in HTML5 where XHTML isn't being used.

Reply:
Very interesting! Thanks for sharing, @TravelTimN! (and yes I am that old that I come from an XHTML background)